pkg/compare/compare_test.go

```diff
@@ -329,6 +333,21 @@ error code:2`),
 			outputFormat: Json,
 			checks:       defaultChecks,
 		},
+		{
+			name:   "Allow Undefined Extras",
+			mode:   []Mode{DefaultMode},
+			checks: defaultChecks,
+		},
+		{
+			name:   "Check No Overwrite",
+			mode:   []Mode{DefaultMode},
+			checks: defaultChecks,
+		},
+		{
+			name:   "Check Probes Merge",
```

Comment:
what does this test check?

Reply:
I don't think this test is needed that was sanity check for me to check the merge would handle strange cases correctly.